resources/views/components/profile/card.blade.php

```diff
@@ -18,6 +18,27 @@
                 {{ $user->name }}
             </a>
         </h1>
+        <div class="flex justify-end">
+            <form method="POST" action="{{ route('profile.follow', $user->id) }}" class="text-right">
```

Comment:
Looks good. in the future, what are ways you could determine the relation status so that you can show/hide the buttons based on if the user has followed?